docs/zh_cn/user_guides/2_dataset_prepare.md

```diff
@@ -38,51 +38,51 @@ mmselfsup
 
 ```
 
-## Prepare ImageNet
+## 准备 ImageNet 数据集
 
-For ImageNet, it has multiple versions, but the most commonly used one is [ILSVRC 2012](http://www.image-net.org/challenges/LSVRC/2012/). It can be accessed with the following steps:
+对于 ImageNet，它有多个版本，但最常用的是 [ILSVRC 2012](http://www.image-net.org/challenges/LSVRC/2012/)。可以通过以下步骤得到：
 
-1. Register an account and login to the [download page](http://www.image-net.org/download-images)
-2. Find download links for ILSVRC2012 and download the following two files
+1. 注册账号并登录 [下载页面](http://www.image-net.org/download-images)
+2. 找到 ILSVRC2012 的下载链接，下载以下两个文件
    - ILSVRC2012_img_train.tar (~138GB)
    - ILSVRC2012_img_val.tar (~6.3GB)
-3. Untar the downloaded files
-4. Download meta data using this [script](https://github.com/BVLC/caffe/blob/master/data/ilsvrc12/get_ilsvrc_aux.sh)
+3. 解压下载的文件
+4. 使用这个 [脚本](https://github.com/BVLC/caffe/blob/master/data/ilsvrc12/get_ilsvrc_aux.sh) 下载元数据
 
-## Prepare Place205
+## 准备 Places205 数据集
 
-For Places205, you need to:
+对于 Places205，您需要：
 
-1. Register an account and login to the [download page](http://places.csail.mit.edu/downloadData.html)
-2. Download the resized images and the image list of train set and validation set of Places205
-3. Untar the downloaded files
+1. 注册账号并登录 [下载页面](http://places.csail.mit.edu/downloadData.html)
+2. 下载 Places205 经过缩放的图片以及训练集和验证集的图片列表
+3. 解压下载的文件
 
-## Prepare iNaturalist2018
+## 准备 iNaturalist2018 数据集
 
-For iNaturalist2018, you need to:
+对于 iNaturalist2018，您需要：
 
-1. Download the training and validation images and annotations from the [download page](https://github.com/visipedia/inat_comp/tree/master/2018)
-2. Untar the downloaded files
-3. Convert the original json annotation format to the list format using the script `tools/dataset_converters/convert_inaturalist.py`
+1. 从 [下载页面](https://github.com/visipedia/inat_comp/tree/master/2018) 下载训练集和验证集图像及标注
+2. 解压下载的文件
+3. 使用脚本 `tools/data_converters/convert_inaturalist.py` 将原来的 json 标注格式转换为列表格式
 
-## Prepare PASCAL VOC
+## 准备 PASCAL VOC 数据集
 
-Assuming that you usually store datasets in `$YOUR_DATA_ROOT`. The following command will automatically download PASCAL VOC 2007 into `$YOUR_DATA_ROOT`, prepare the required files, create a folder `data` under `$MMSELFSUP` and make a symlink `VOCdevkit`.
+假设您通常将数据集存储在 `$YOUR_DATA_ROOT` 中。下面的命令会自动将 PASCAL VOC 2007 下载到 `$YOUR_DATA_ROOT` 中，准备好所需的文件，在 `$MMSELFSUP` 下创建一个文件夹 `data`，并制作一个软链接 `VOCdevkit`。
 
 ```shell
 bash tools/dataset_converters/prepare_voc07_cls.sh $YOUR_DATA_ROOT
 ```
 
-## Prepare CIFAR10
+## 准备 CIFAR10 数据集
 
-CIFAR10 will be downloaded automatically if it is not found. In addition, `dataset` implemented by `MMSelfSup` will also automatically structure CIFAR10 to the appropriate format.
+如果没有找到 CIFAR10 系统将会自动下载。此外，由 `MMSelfSup` 实现的 `dataset` 也会自动将 CIFAR10 转换为适当的格式。
```

Comment:
Currently, we don't have CIFAR10 dataset, it is recommended to use `mmcls.CIFAR10` or `mmcls.CIFAR100` in the config. Please add some contents to illustrate this, also modify the English version of the doc simultaneously.